features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Listing/CommonListing.jsx

```diff
@@ -196,6 +198,7 @@ class CommonListing extends React.Component {
      * @memberof CommonListing
      */
     setListType = (value) => {
+        localStorage.setItem('portal.listType', value);
```

Comment:
Shall we set this local storage item through the `User` instance/object ? So that we know when we clear the user all their related local settings get cleared too

Reply:
Have an issue with that approach. There is no user instance for the anonymous user. Any suggestions to handle the anonymous user?

Reply:
ahh good point, Shall we make a special user object for anonymous user ? and store under the same local storage key for `devportal` user. and when a new user logs in, We can override the anonymous user properties with new user's properties. ?